docs/topics/spider-middleware.rst

```diff
@@ -328,6 +328,66 @@ Default: ``True``
 
 Whether to enable referer middleware.
 
+.. setting:: REFERER_POLICY
+
+REFERER_POLICY
+^^^^^^^^^^^^^^
+
+.. versionadded:: 1.4
+
+Default: ``'scrapy.spidermiddlewares.referer.DefaultReferrerPolicy'``
+
+.. reqmeta:: referrer_policy
+
+`Referrer Policy`_ to apply when populating Request "Referer" header.
+
+This setting accepts:
+
+- a path to a ``scrapy.spidermiddlewares.referer.ReferrerPolicy`` subclass,
+  either a custom one or one of the built-in ones
+  (see ``scrapy.spidermiddlewares.referer``),
+- or one of the standard W3C-defined string values:
+
+  - `"no-referrer"`_,
+  - `"no-referrer-when-downgrade"`_,
+  - `"same-origin"`_,
+  - `"origin"`_,
+  - `"strict-origin"`_,
+  - `"origin-when-cross-origin"`_,
+  - `"strict-origin-when-cross-origin"`_,
+  - or `"unsafe-url"`_
+    (not recommended).
+
+  (It can also be the non-standard value ``"scrapy-default"`` to use
+  Scrapy's default referrer policy.)
+
+Scrapy's default referrer policy is a variant of `"no-referrer-when-downgrade"`_,
+with the addition that "Referrer" is not sent if the parent request was
+using ``file://`` or ``s3://`` scheme.
+
+.. warning::
+    Scrapy's default referrer policy, just like `"no-referrer-when-downgrade"`_,
+    will send a non-empty "Referer" header from any ``https://`` to any ``https://`` URL,
+    even if the domain is different.
```

Comment:
If that's what browsers do by default then I think it makes sense to note that, to make a warning less scary.